integ/integ.sh

```diff
@@ -2,7 +2,7 @@
 export AWS_REGION="us-west-2"
 export PROJECT_ROOT="$(pwd)"
 export VOLUME_MOUNT_CONTAINER="/out"
-export ValidateS3Dockerfile="Dockerfile"
+export ValidateS3Dockerfile="./scripts/dockerfiles/Dockerfile"
```

Comment:
I think we should keep the integ dockerfiles inside the integ folder. Also is this actually the path?

Reply:
Also, this is a great catch, thank you!